practice/case.py

```diff
@@ -0,0 +1,55 @@
+""'This program converts text from snake_case to CamelCase and from CamelCase to snake_case""'
+
+# 1. Setup
+import re
+
+# 2. Define
+
+def prompt_for_input_phrase():
+    """Gathers input word from user"""
+    return input('What is the word you would like to transform? ')
+    # return 'test_word'
+
+def detect_case(input_word):
+    """Detects whether the word is in snake_case or not, generates bool value storing this
+    >>> detect_case('test_word')
+    True
+    """
+    is_snake_case = False if input_word[0].isupper() == True else True
```

Comment:
You can just say `is_snake_case = not input_word[0].isupper()`.

You can use boolean return values directly.